generator/recipe.go

```diff
@@ -30,43 +31,39 @@ func Recipe(name string, source string, sinks []string, processors []string) (er
 
 	if source != "" {
 		tem.Source = make(map[string]string)
-		sinfo, err := registry.Extractors.Info(source)
+		sourceInfo, err := registry.Extractors.Info(source)
 		if err != nil {
-			return err
+			return errors.Wrapf(err, "failed to provide extractor information")
```

Comment:
same here, we use `Wrapf` if there is string formatting, we can use `Wrap` instead.